index.rst

```diff
@@ -10,7 +10,240 @@ This is complemented by the so called "run branch", which is a common branch use
+
+.. prompt:: bash
+
+   sudo -iu obsenv
+
+Then, you can run ``manage_obs_env`` to perform operations on the observing environment.
+
+This cli supports a number of operations to manage the version of the repositories that are part of the observing environment.
+The most common user operations revolve around the run branch and includes:
+
+- Checkout the run branch for a particular repository:
+
+  .. prompt:: bash
+
+    manage_obs_env --action checkout-run-branch --repository <repository-name>
+
+- List run branch:
+
+  .. prompt:: bash
+
+    manage_obs_env --action list-run-branch
+
+  This will print to stdout the name of the run branch.
+
+- Checkout a specific branch for a particular repository:
+
+  .. prompt:: bash 
+    
+    manage_obs_env --action checkout-branch --repository <repository-name> --branch-name <branch-name>
+
+- Checkout a specific version for a particular repository.
+
+  .. prompt:: bash 
+    
+    manage_obs_env --action checkout-version --repository <repository-name> --branch-name <version>
+
+- Show current version of the packages:
+
+  .. prompt:: bash
+
+    manage_obs_env --action show-current-versions
+
+More advanced operations 
+
+- Register a new run branch:
+  
+  .. prompt:: bash
+
+    manage_obs_env --action register-run-branch --branch-name <branch-name>
+
+  This operation is usually only made once at the start of the run and only need to be repeated when a new run branch is created.
+
+- Reset the observing environment:
+
+  .. prompt:: bash
+
+    manage_obs_env --action reset
+
+  This operation will reset the version of all repositories to their default version.
+  The default version is either the version specified in the cycle build file or, if the repository has a run branch, the run branch.
+
+The ``manage_obs_env`` has a number of other operations that but that are beyong the scope of this document.
+
+Web Interface
+-------------
+
+For users that prefer graphical interface, there is a web interface available to manage the observing environment.
+These can be accessed at the following urls:
+
+- Summit: https://summit-lsp.lsst.codes/obsenv-management/dashboard
+- BTS: https://base-lsp.lsst.codes/obsenv-management/dashboard
+
+.. _traceability:
+
+Traceability
+------------
+
+One of the key features of the ``manage_obs_env`` application is that it allows us to trace the user actions and the state of the observing environment.
+Whenever a user runs the ``manage_obs_env`` it will publish the action performed, alongside its parameters, time and the user name of who performed it.
+It also published a summary of the repositories version after the action is performed.
+
+The information is readily available in sasquatch (:sqr:`068`) under the ``obsenv`` database, and can be inspected using Chronograf.
+
+Observing Environment Repositories
+----------------------------------
+
+At the time of this writting the following repositories are part of the observing environment:
```

Comment:
Maybe we can add a link to (https://github.com/lsst-ts/ts_manage_observing_environment/blob/main/src/repos.rs)